studio/components/layouts/ProjectLayout/LayoutHeader/HelpPopover.tsx

```diff
@@ -62,6 +63,15 @@ const HelpPopover: FC<Props> = () => {
               Expected response time is based on your billing plan. Pro and Pay as You Go plans are
               prioritized.
             </p>
+            <div>
+              <Link passHref href={supportUrl}>
+                <a>
+                  <Button type="default" icon={<IconMail />}>
+                    Contact Support
+                  </Button>
+                </a>
+              </Link>
```

Comment:
Can we change these to `asChild` buttons to avoid the `<button />` inside the `<a />`?

cc @MildTomato `asChild` works on our current buttons right?

Reply:
correct composition is 
```
<Link passhref>
<Button aschild>
<a/>
</Button>
</Link>
```